app/Global.scala

```diff
@@ -73,65 +76,89 @@ object Global extends GlobalSettings with LazyLogging{
  * Initial set of data to be imported
  * in the sample application.
  */
-object InitialData extends GlobalDBAccess with LazyLogging {
-
-  val mpi = Team("Connectomics department", None, RoleService.roles)
-
-  def insert(conf: Configuration) = {
-    insertDefaultUser(conf)
-    insertTeams()
-    insertTasks()
-    if (conf.getBoolean("datastore.enabled").getOrElse(true)) {
-      insertLocalDataStore(conf)
-    }
-  }
-
-  def insertDefaultUser(conf: Configuration) = {
+object InitialData extends GlobalDBAccess with FoxImplicits with LazyLogging {
+
+  val defaultUserEmail = Play.configuration.getString("application.authentication.defaultUser.email").get
```

Comment:
What happens, if the key ist not set in the config? Probably an exception. Is that what we want here (not a rhetorical question)

Reply:
I re-added the getOrElse here for now to keep things consistent.
There is a general discussion on config fallback values in issue #2326 , which would also love your input :)